hare/consensus_test.go

```diff
@@ -112,7 +112,7 @@ func (his *HareSuite) checkResult(t *testing.T) {
 	}
 
 	// check that the output has no intersection with the complement of the union of honest
-	for _, v := range his.outputs[0].elements() {
+	for _, v := range his.outputs[0].ToSlice() {
```

Comment:
This iterates twice! Once over all elements of `outputs` to get the values and once again over the values.